test/integration/sonobuoy_integration_test.go

```diff
@@ -443,3 +446,44 @@ func pwd(t *testing.T) string {
 	}
 	return pwd
 }
+
+// TestExactOutput is to test things which can expect exact output; so do not use it
+// for things like configs which include timestamps or UUIDs.
+func TestExactOutput(t *testing.T) {
```

Comment:
Will be quite useful testing other commands. We have goldenfiles for some gen logic but using it based on just cmdline logic is way more wholistic AND simple.